pkg/client/testdata/systemd-logs-default.golden

```diff
@@ -78,6 +83,9 @@ data:
             fieldPath: spec.nodeName
       - name: RESULTS_DIR
         value: /tmp/results
+      - name: SONOBUOY_K8S_VERSION
+        value: v99+static.testing
```

Comment:
Lots of lines like this; before the tests would take an empty k8s version and do nothing with it at all which was generating goldenfiles that were slightly misleading. They wouldn't have default images defined and wouldn't set the k8s_version env var as any other `gen` run would.